.github/workflows/dockerimage-clang-11-ubuntu-mpich-vt.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       MAGISTRATE_MPI: 1
       MAGISTRATE_WARNINGS_AS_ERRORS: 1
       MAGISTRATE_DOCS: 0
-      MAGISTRATE_ASAN: 1
-      MAGISTRATE_UBSAN: 1
+      MAGISTRATE_ASAN: 0
+      MAGISTRATE_UBSAN: 0
```

Comment:
TODO: create a follow-up issue. Decide whether asan + ubsan combo is necessary (maybe we can skip ubsan here to avoid issues).